GitUp/Application/AppDelegate.m

```diff
@@ -335,6 +335,8 @@ - (void)applicationWillFinishLaunching:(NSNotification*)notification {
   [Crashlytics startWithAPIKey:@"946dcb56f0db1be8f0c52df6e7d392202a2cc9b2"];
   [[Crashlytics sharedInstance] setDelegate:self];
 #endif
+  
+  [[NSAppleEventManager sharedAppleEventManager] setEventHandler:self andSelector:@selector(_getUrl:withReplyEvent:) forEventClass:kInternetEventClass andEventID:kAEGetURL];
```

Comment:
It means if I put it in `-applicationDidFinishLaunching:` it wouldn't trigger any event if I GitUp wasn't open yet.